fri/src/two_adic_pcs.rs

```diff
@@ -431,6 +442,7 @@ where
             // `reduced_openings` would have a log_height = log_blowup entry only if there was a
             // trace matrix of height 1. In this case the reduced opening can be skipped as it will
             // not be checked against any commit phase commit.
+            #[cfg(debug_assertions)]
```

Comment:
Hm would this affect the height 1 edge case? cc @jonathanpwang who added this (https://github.com/Plonky3/Plonky3/pull/415)

If the issue was the unused var in release mode, it probably wouldn't hurt to use `assert!` below

Reply:
Yes this should only be `Some` if there was an unused `ro` and it's a sanity check that `ro == 0`. I agree we can change it to an `assert!` for safety